mimic/rest/rackconnect_v3_api.py

```diff
@@ -0,0 +1,447 @@
+# -*- test-case-name: mimic.test.test_rackconnect_v3 -*-
+
+"""
+API mock for the Rackspace RackConnect v3 API, which is documented at:
+http://docs.rcv3.apiary.io/
+"""
+from collections import defaultdict
+import json
+from uuid import uuid4, UUID
+
+from characteristic import attributes, Attribute
+from six import text_type
+
+from twisted.plugin import IPlugin
+from twisted.web.http import NOT_FOUND, NOT_IMPLEMENTED
+from twisted.web.server import Request
+from zope.interface import implementer
+
+from mimic.catalog import Entry
+from mimic.catalog import Endpoint
+from mimic.imimic import IAPIMock
+from mimic.rest.mimicapp import MimicApp
+from mimic.util.helper import random_ipv4, seconds_to_timestamp
+
+
+Request.defaultContentType = 'application/json'
+timestamp_format = '%Y-%m-%dT%H:%M:%SZ'
+
+
+@implementer(IAPIMock, IPlugin)
+class RackConnectV3(object):
+    """
+    API mock object for RackConnect V3.
+
+    :ivar regions: The regions for which endpoints should be produced
+    :type regions: ``iterable``
+
+    :ivar int default_lbs: The number of default load balancers to be created
```

Comment:
The terminology is confusing, and AFAICT there's no real difference between "load balancer" and "load balancer pool" (and the latter sounds like a bunch of LBs even though it isn't). Have you considered using the RCv3 terminology here?